### PR TITLE
Expand search examples from ailab-db (PostgreSQL)

### PR DESCRIPTION
**Pull request by melanie-fressard**
issue #49
issue #48

### CODE REVIEW

**Review by k-allagbe — CHANGES_REQUESTED** [review]
Don't forget to address the original issue in `finesse-data` and the lint fail.

File: tests/output/How-do-I-use-digital-certificates.json [hunk 1]
```diff
@@ -0,0 +1 @@
...our identity information and click Continue fill in your name and email address from the Key Algorithm menu, choose an option (the 2048-bit RSA option offers more security than 1024-bit RSA, but 1024-bit RSA is more universally compatible) from the Use Digital ID For menu, choose whether you want to use the Digital ID for signatures, data encryption, or both click Browse to select where you would like to save the Digital ID on your device or use the default location as shown in the File Name field remember where you saved the Digital ID so that you can copy it or back it up enter and confirm a password to protect your Digital ID and click Save you will need this password when you sign a document using your Digital ID Sign a form using a Digital ID open the PDF form you wish to sign and click the Signature box choose the Digital ID that you want to use for signing and click Continue enter the password for your Digital ID and click Sign save the completed form to your device remember where you saved the form so you can access it later", "query_id": "759104ab-013f-459a-b01d-165401ddcf03", "subtitle": "How to sign a form digitally", "last_updated": "2018-12-13", "tokens_count": 330}]}]
```
[the remark this answers is not in the document]
Comment (melanie-fressard) [reply tests/output/How-do-I-use-digital-certificates.json]: done

File: tests/output/how-to-bring-a-cat-from-france-to-canada.json [hunk 2]
```diff
@@ -0,0 +1 @@
...075}, "content": "Export of Dogs and Cats to Saint-Pierre and Miquelon Saint-Pierre and Miquelon is a chain of islands south of Newfoundland. It is a self-governing territorial overseas collectivity of France. Importation of animals in Saint-Pierre and Miquelon is strictly regulated. The requirements are as follows: The animal must be 4 months or older. The animal must be vaccinated for rabies and have proper certificates from a licensed veterinarian (If it's the animal's first vaccination, the animal must wait a month before travelling. If it's just the yearly rabies vaccination, the animal can travel right away). The animal must have a visible tattoo or have a microchip implanted with the proper certification. The animal must have a \"good bill of health\" certificate from a licensed veterinarian certifying that the animal is free from disease. The certificate should be issued a maximum of 4 days prior to travelling. There is no CFIA certification requirement for export of pets to Saint-Pierre and Miquelon.", "query_id": "77ae95d8-9125-4c31-81ad-f85563cdb868", "subtitle": "Export of Dogs and Cats to Saint-Pierre and Miquelon", "last_updated": "2016-10-05", "tokens_count": 206}]}]
```
[the remark this answers is not in the document]
Comment (melanie-fressard) [reply tests/output/how-to-bring-a-cat-from-france-to-canada.json]: done

File: ailab/db/api/__init__.py [hunk 3]
```diff
@@ -1,8 +1,8 @@
-import louis.models.openai as openai
+import ailab.models.openai as openai
 import os
 import json
 import sys
-import louis.db as db
+import ailab.db as db
```
[the remark this answers is not in the document]
Comment (melanie-fressard) [reply ailab/db/api/__init__.py:5]: done

File: setup-procedure [hunk 4]
```diff
@@ -8,9 +8,9 @@ Once you're on VS Code, you have to do the following :
 - Type "cd" to return to your $HOME folder.
```
Comment (k-allagbe) [inline setup-procedure:8]: This file doesn't have an extension.

File: bin/search.py [hunk 5]
```diff
@@ -1,8 +1,9 @@
 import subprocess
 import sys
+import json
 import logging
-import louis.db.api as api
-import louis.db as db
+import ailab.db.api as api
+import ailab.db as db
 from microbench import MicroBench, MBReturnValue, MBFunctionCall
 import pandas as pd
```
Comment (k-allagbe) [inline bin/search.py:8]: The lint fail point to this line with message: "bin/search.py:8:18: F401 `pandas` imported but unused". You can also check the reason for failure in the repo's "action" tab: https://github.com/ai-cfia/ailab-db/actions/runs/6945568744/job/18895234952?pr=50

**Review by k-allagbe — CHANGES_REQUESTED** [review]

File: bin/search.py (hunk 5 above)
Comment (melanie-fressard) [reply bin/search.py]: forgot about that, done

**Review by k-allagbe — APPROVED** [review]

File: bin/search.py [hunk 6]
```diff
@@ -27,12 +25,12 @@ def init_bench(query):
...
 if __name__ == "__main__":
     """Execute the sql search function with the query passed as an argument. 
     How to execute: python3 -m search.py query"""
     query = " ".join(sys.argv[1:])  
     logging.debug(f"Query is: {query}")
-    init_bench(query)
-    with open(OUTFILE, "r") as result_file:
-        print(pd.read_json(result_file, lines=True))
+    results = init_bench(query)
+    query_file_name = query.replace (" ", "-") + ".json"
```
Comment (rngadam) [inline bin/search.py:34]: https://nadeauinnovations.com/post/2020/11/python-tricks-replace-all-non-alphanumeric-characters-in-a-string/

**Review by rngadam — APPROVED** [review]

File: bin/search.py (hunk 6 above)
Comment (melanie-fressard) [reply bin/search.py]: keeping the spaces as it is made for finesse-frontend

File: tests/output/How do I use digital certificates.json [hunk 7]
```diff
@@ -0,0 +1,185 @@
+[
+    {
+        "search": [
```
Comment (k-allagbe) [inline tests/output/How do I use digital certificates.json:3]: The format is different from before. 
Desired:
```json
[
    {},
    {}
]
```
Currently:
```json
[
   {
        "search": [
            {},
            {}
        ]
   }
]
```

**Review by k-allagbe — CHANGES_REQUESTED** [review]

File: tests/output/How do I use digital certificates.json (hunk 7 above)
Comment (melanie-fressard) [reply tests/output/How do I use digital certificates.json]: done

**Review by k-allagbe — APPROVED** [review]

**Pull request merged**